common/test/acceptance/tests/lms/test_lms_user_preview.py

```diff
@@ -383,6 +382,24 @@ def test_as_specific_student(self):
         course_page.set_staff_view_mode_specific_student(student_b_username)
         verify_expected_problem_visibility(self, course_page, [self.beta_text, self.everyone_text])
 
+    @attr('a11y')
+    def test_course_page(self):
+        """
+        Run accessibility audit for course staff pages.
+        """
+        course_page = self._goto_staff_page()
+        course_page.a11y_audit.config.set_rules({
+            'ignore': [
+                'aria-allowed-attr',  # TODO: AC-559
+                'aria-roles',  # TODO: AC-559,
+                'aria-valid-attr',  # TODO: AC-559
+                'color-contrast',  # TODO: AC-559
+                'link-href',  # TODO: AC-559
```

Comment:
This is for staff privileges (previewing as a different role or student) so I excluded it. These previews/modals need a great deal of work and should be its own ticket (AC-559).